### PR TITLE
113 implement settings page frontend

### PR DESCRIPTION
**Pull request by erenfn**
image
image
image
solves #113

### CODE REVIEW

**Review by uparkalau — COMMENTED** [review]
Review Coderabbit suggestions. Refactor HTML and CSS styling with variables. Remove code duplication, implement reusable components where possible.

**Comment by erenfn** [comment]
> Review Coderabbit suggestions. Refactor HTML and CSS styling with variables. Remove code duplication, implement reusable components where possible.

done

**Review by uparkalau — APPROVED** [review]

File: frontend/src/scenes/settings/TeamTab/TeamTab.module.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+.normalTab {
+    font-weight: normal;
+}
+
+.tabIndicator {
+    background-color: rgba(244, 245, 247, 1);
+    padding-top: 0px;
+}
```
[the remark this answers is not in the document]
Comment (uparkalau) [reply frontend/src/scenes/settings/TeamTab/TeamTab.module.css:50]: Check it before merging

File: frontend/src/scenes/settings/TeamTab/TeamTab.module.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    margin: 0;
+}
+
+.organisationName {
+    margin: 0;
+    display: flex;
+    align-items: center;
+    gap: 0.6rem;
+    font-size: 13px;
+}
```
[the remark this answers is not in the document]
Comment (uparkalau) [reply frontend/src/scenes/settings/TeamTab/TeamTab.module.css:19]: Add new variable

File: frontend/src/scenes/settings/TeamTab/TeamTab.module.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
+.organisation {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid rgba(238, 238, 238, 1);
+    padding-bottom: 15px;
+}
```
[the remark this answers is not in the document]
Comment (uparkalau) [reply frontend/src/scenes/settings/TeamTab/TeamTab.module.css:7]: You can implement this variables as well

File: frontend/src/scenes/settings/TeamTab/TeamTab.module.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+h6 {
+    font-size: 13px;
+    font-weight: 600;
+}
+
+.MuiTabPanel-root {
+    padding: 0
+}
```
[the remark this answers is not in the document]
Comment (uparkalau) [reply frontend/src/scenes/settings/TeamTab/TeamTab.module.css:37]: Create a new issue to implement this suggestion

File: frontend/src/scenes/settings/TeamTab/TeamTab.module.css [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+.tabs {
+    border: 1px solid rgba(234, 236, 240, 1) !important;
+}
+
+h6 {
+    font-size: 13px;
+    font-weight: 600;
+}
```
[the remark this answers is not in the document]
Comment (uparkalau) [reply frontend/src/scenes/settings/TeamTab/TeamTab.module.css:33]: Check it

File: frontend/src/scenes/settings/TeamTab/TeamTab.module.css [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+.team {
+    display: flex;
+    justify-content: space-between;
+}
+
+.tabs {
+    border: 1px solid rgba(234, 236, 240, 1) !important;
+}
```
[the remark this answers is not in the document]
Comment (uparkalau) [reply frontend/src/scenes/settings/TeamTab/TeamTab.module.css:28]: Double check variables implementation

File: frontend/src/scenes/settings/TeamTab/TeamTab.module.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (erenfn) [reply frontend/src/scenes/settings/TeamTab/TeamTab.module.css:19]: We'll refactor all font-sizes to a variable in a seperate issue

**Pull request merged**